Feat/lost person create

abdulrahman-badry777 opened the pull request:
## Description

I worked on the `lost_person_case` and added the pages available on Figma to allow the user to add a new missing person case and input the necessary data dynamically. 

I removed all hard-coded data being sent to the API and made it dynamic to send what the user entered. 

Additionally, I recorded a video for the task, and here is the link:  
[Task Video](https://drive.google.com/file/d/1DeI13jjaNJTNeLEBBGj0KYzpjMmvaNTr/view?usp=drive_link)

I used `Vue.js` for all active actions.

Please review the changes and let me know if any further adjustments are needed.

Thank you!

Al0olo reviewed (changes requested):
After Submitting the form don't forget to clear the localstorage.

abdulrahman-badry777 commented:
> After Submitting the form don't forget to clear the localstorage.



> After Submitting the form don't forget to clear the localstorage.



> ## Description
> I worked on the `lost_person_case` and added the pages available on Figma to allow the user to add a new missing person case and input the necessary data dynamically.
> 
> I removed all hard-coded data being sent to the API and made it dynamic to send what the user entered.
> 
> Additionally, I recorded a video for the task, and here is the link: [Task Video](https://drive.google.com/file/d/1DeI13jjaNJTNeLEBBGj0KYzpjMmvaNTr/view?usp=drive_link)
> 
> I used `Vue.js` for all active actions.
> 
> Please review the changes and let me know if any further adjustments are needed.
> 
> Thank you!



> After Submitting the form don't forget to clear the localstorage.



> After Submitting the form don't forget to clear the localstorage.

abdulrahman-badry777 commented:
> After Submitting the form don't forget to clear the localstorage.

Thank you for your feedback!

I have already implemented the code to clear the **localStorage** after submitting the form in the `index.js` file located in the `lost_person_create` folder. Please let me know if there's anything else that needs adjustment or improvement.
